pages/products.yaml

```diff
@@ -121,7 +121,7 @@ events:
 areas:
   header:
     blocks:
-      - _ref: view_github.yaml
+      - _ref: shared/components/view_github.yaml
   content:
     blocks:
       - id: title # Create a Title block
```

Comment:
Maybe this could be split into `products_list`, `new_product_drawer`, `edit_product_drawer` and `delete_product_drawer`

Reply:
Change made in a487637cbd263686e96adb2a213075a057070e2f